stable/mongodb-replicaset/README.md

```diff
@@ -89,6 +92,72 @@ parameter `auth.enabled`. Once enabled, keyfile access control is set up and an
 is created. User credentials and keyfile may be specified directly. Alternatively, existing secrets may be provided. 
 The secret for the admin user must contain the keys `user` and `password`, that for the key file must contain `key.txt`.
 
+## TLS support
+
+To enable full TLS encryption setl `tls.enabled` to `true`. It is recommended to create your own CA by executing:
+
+```console
+$ openssl genrsa -out ca.key 2048
+$ openssl req -x509 -new -nodes -key ca.key -days 10000 -out ca.crt -subj "/CN=mydomain.com"
+```
+
+After that paste the base64 encoded (`cat ca.key | base64 -w0`) cert and key into the fields `tls.cacert` and
+`tls.cakey`. Adapt the configmap for the replicaset as following:
+
+```yml
+configmap:
+  storage:
+    dbPath: /data/db
+  net:
+    port: 27017
+    ssl:
+      mode: requireSSL
+      CAFile: /ca/tls.crt
+      clusterFile: /work-dir/mongo.pem
```

Comment:
What is `clusterFile` for?

Reply:
`clusterFile` is the internal auth file for the replicaset. I guess a fallback is the pemkeyfile, so I can remove the clusterfile there.